akka-distributed-data/src/main/scala/akka/cluster/ddata/DeltaPropagationSelector.scala

```diff
@@ -104,7 +105,13 @@ import akka.cluster.ddata.Replicator.Internal.DeltaPropagation
               val deltaGroup = cache.get(cacheKey) match {
                 case None ⇒
                   val group = deltaEntriesAfterJ.valuesIterator.reduceLeft {
-                    (d1, d2) ⇒ d1.merge(d2.asInstanceOf[d1.T])
+                    (d1, d2) ⇒
+                      d2 match {
+                        case NoDeltaPlaceholder ⇒ NoDeltaPlaceholder
```

Comment:
this is when grouping together deltas before sending them, if there is one such None in the group the whole group is "invalid"